zerver/lib/email_notifications.py

```diff
@@ -384,10 +384,8 @@ def do_send_missedmessage_events_reply_in_zulip(user_profile: UserProfile,
         # Keep only the senders who actually mentioned the user
         if context['mention']:
             senders = list(set(m['message'].sender for m in missed_messages
-                           if m['trigger'] == 'mentioned'))
-            # TODO: When we add wildcard mentions that send emails, we
-            # should make sure the right logic applies here.
-
+                               if m['trigger'] == 'mentioned' or
+                               m['trigger'] == 'wildcard_mentioned'))
```

Comment:
This is the block we need to change if we want wildcard mentions to have different styling from normal mentions in email notifications.  My current view is that I don't know of better text we can use than "because you were mentioned", so there's no need, but FYI.

@rishig FYI.

Reply:
Yeah, I think the current default is fine, and we can update it after seeing in the wild.